letsencrypt/client/configuration.py

```diff
@@ -48,6 +48,18 @@ def cert_key_backup(self):  # pylint: disable=missing-docstring
             self.namespace.work_dir, constants.CERT_KEY_BACKUP_DIR,
             self.namespace.server.partition(":")[0])
 
+    @property
+    def accounts_dir(self):  #pylint: disable=missing-docstring
+        return os.path.join(
+            self.namespace.config_dir, constants.ACCOUNTS_DIR,
+            self.namespace.server.partition(":")[0])
```

Comment:
Also, this will have to necessarily be fixed after `--server https://www.letsencrypt-demo.org` change.